src/style.css

```diff
@@ -1,14 +1,20 @@
 
 @import url('https://fonts.googleapis.com/css2?family=Poppins:wght@500;700&display=swap');
 
+:root {
+  --primary-color: #0a0add;
+  --light-color: #ccc;
+  --white-color: #fff;
+}
+
```

Comment:
- [ ] You have done a great job so far. :+1: :+1:  But I think there are still some styling issues you need to fix before moving to the next milestone.
- - Looking at the video below, you can see that your score list overflows the page and users can't see the website title or the refresh button, THis does not match the given design and also is not good for user experience. I'll suggest fixing this. You can try giving the score table a height and adding a vertical, scroll for the overflowing content. :+1: :100: 

- - VIDEO FOR REFERENCE:

https://github.com/ghermaico135/loaderboard-list/assets/107632780/de758974-d65c-4762-948c-185cb00b2afd